### PR TITLE
Add Artificial General Intelligence

### PR DESCRIPTION
**Pull request by razvan-panda**
https://github.com/fairy-tale-agi-solutions/awesome-artificial-general-intelligence

Artificial general intelligence (AGI) is the intelligence of a machine that could successfully perform any intellectual task that a human being can. It is a primary goal of some artificial intelligence research and a common topic in science fiction and future studies. Artificial general intelligence is also referred to as "strong AI", "full AI" or as the ability of a machine to perform "general intelligent action".

Pull requests I reviewed:

https://github.com/sindresorhus/awesome/pull/1407
https://github.com/sindresorhus/awesome/pull/1420
https://github.com/sindresorhus/awesome/pull/1415
https://github.com/sindresorhus/awesome/pull/1414

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).** Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- I have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title in the format `Add Name of List` (Example: `Add Swift`), not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of sub-lists, preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -318,6 +318,7 @@
 - [Algorithms](https://github.com/tayllan/awesome-algorithms#readme)
 - [Algorithm Visualizations](https://github.com/enjalot/algovis#readme)
 - [Artificial Intelligence](https://github.com/owainlewis/awesome-artificial-intelligence#readme)
+- [Artificial General Intelligence](https://github.com/fairy-tale-agi-solutions/awesome-artificial-general-intelligence#readme)
```
[the remark this answers is not in the document]
Comment (razvan-panda) [reply readme.md]: Added

**Comment by razvan-panda** [comment]
@sindresorhus I added the description.

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/fairy-tale-agi-solutions/awesome-artificial-general-intelligence
✖ Linting

  README.md:1:1
  ✖   1:1   Missing newline character at end of file                                                final-newline
  ✖  15:1   ToC item "Forums / News" does not match corresponding heading "Forums / News / Social"  awesome/toc
  ✖  54:3   Don’t pad link with inner spaces                                                        no-inline-padding
  ✖  98:42  List item description must end with proper punctuation                                  awesome/list-item
```

https://github.com/sindresorhus/awesome-lint


---

Linkify `Artificial general intelligence` in `Artificial general intelligence (AGI) is the intelligence of a machine that could successfully perform any` to the Wikipedia page: https://en.wikipedia.org/wiki/Artificial_general_intelligence


---

There are some items that are missing a description where it could be useful. Items in the `Podcasts`, `Videos`, `Organizations / Projects`, `Experts on Twitter`, `Movies`, `TV Shows`, and `Anime` sections.

---

The heading should be `Awesome Artificial General Intelligence`, not `Awesome resources about Artificial General Intelligence`.

**Comment by razvan-panda** [comment]
@sindresorhus Thank you for reviewing.

I made the changes requested and now the lint is passing.

**Comment by sindresorhus** [comment]
The entries in the `Movies`, `TV Shows`, and `Anime` section could use a description with a short summary.

**Comment by sindresorhus** [comment]
For entries in the `Experts on Twitter` section, it would be useful to describe **what** makes them experts on this.

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**